src/LessMsi.Gui/MainForm.cs

```diff
@@ -184,13 +184,43 @@ public void SetTableViewGridDataSource(IEnumerable<object[]> values)
 			{
 				msiTableGrid.Rows.Add(row);
 			}
+			if (msiTableGrid.IsHandleCreated)
+			{
+				msiTableGrid.AutoResizeColumns();

```

Comment:
Another option would be to set the property for this, but that slows down scrolling in the control.

Reply:
If the slowdown isn't significant I prefer the property to reduce the overall code complexity. If the slowdown is significant (use your judgement on "significant"), I'm fine with finding a place to make the explicit method call. Seems unnatural to have to add such an event handler just to get the columns to sort, but if that's the way it is 🤷

Reply:
The property was the first thing I did, but the scrolling was just annoying enough for me to spend the time to do it this way.